contrib/profiler/Profiler.h

```diff
@@ -11,7 +11,7 @@
 #undef fastcall
 
 //#define USE_CHRONO
-#if !defined(USE_CHRONO) && defined(__arm__)
+#if !defined(USE_CHRONO) && defined(__arm__) || defined(__aarch64__)
```

Comment:
This part is not correct, in my opinion. It should be:
`#if !defined(USE_CHRONO) && (defined(__arm__) || defined(__aarch64__))`